app/src/main/res/values/strings.xml

```diff
@@ -102,50 +102,61 @@
+  <string name="admin_pin_new_pin">New 5-Digit PIN</string>
+  <string name="admin_pin_new_confirm">Confirm 5-Digit PIN</string>
+  <string name="admin_pin_error_pin_length">Your PIN should be 5 digits long.</string>
+  <string name="admin_pin_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
+  <!-- AddProfileActivity -->
   <string name="add_profile_title">Add Profile</string>
   <string name="add_profile_input_name">Name (required)</string>
   <string name="add_profile_input_pin">3-Digit PIN (optional)</string>
   <string name="add_profile_input_confirm_pin">Confirm 3-Digit PIN</string>
   <string name="add_profile_allow_download_heading">Allow Download Access</string>
   <string name="add_profile_allow_download_sub">User is able to download and delete content without Administrator password. Only available with PIN.</string>
-  <string name="add_profile_create_btn">create</string>
-  <string name="add_profile_close">CLOSE</string>
+  <string name="add_profile_create_btn">Create</string>
+  <string name="add_profile_close">Close</string>
   <string name="add_profile_pin_info">With a PIN, nobody else can access a profile besides this assigned user.</string>
   <string name="add_profile_error_image_failed_store">We failed to store your avatar image. Please try again.</string>
   <string name="add_profile_error_name_not_unique">This name is already in use by another profile.</string>
   <string name="add_profile_error_name_empty">Please enter a name for this profile.</string>
   <string name="add_profile_error_name_only_letters">Names can only have letters. Try another name?</string>
   <string name="add_profile_error_pin_length">Your PIN should be 3 digits long.</string>
   <string name="add_profile_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
-  <!-- PinPasswordActivity. -->
+  <!-- PinPasswordActivity -->
   <string name="pin_password_hello">Hi, %s!</string>
   <string name="pin_password_admin_enter">Please enter your \nAdministrator PIN</string>
   <string name="pin_password_user_enter">Please enter your PIN</string>
   <string name="pin_password_forgot_pin">I forgot my pin</string>
   <string name="pin_password_incorrect_pin">Incorrect PIN.</string>
   <string name="pin_password_show">show</string>
   <string name="pin_password_hide">hide</string>
-  <string name="pin_password_close">CLOSE</string>
+  <string name="pin_password_close">Close</string>
   <string name="pin_password_success">PIN change is successful</string>
   <string name="pin_password_forgot_title">Forgot PIN?</string>
   <string name="pin_password_forgot_message">To reset your PIN, please uninstall Oppia and then reinstall it.\n\nKeep in mind that if the device has not been online, you may lose user progress on multiple accounts. </string>
-  <string name="pin_password_play_store">GO TO PLAY STORE</string>
-  <!-- AdminSettingsDialogFragment. -->
+  <string name="pin_password_play_store">Go to the play store</string>
```

Comment:
Why does the forgot pin flow navigate the user to the play store?

Reply:
That is the flow that was in the mocks if the user is the admin.

Reply:
Ack--that actually confuses me but I can follow up with Chantel.